pkg/controller/zerocapacity/zero_controller.go

```diff
@@ -102,35 +102,23 @@ const (
 	ZeroUnknown Phase = "Unknown"
 )
 
-func CreateController(name string, reconciler Reconciler, mgr manager.Manager) error {
+func CreateController(name string, reconciler Reconciler, mgr ctrl.Manager, eventRec record.EventRecorder) error {
```

Comment:
`zero_controller.go` should be moved to the `controllers` dir.

Reply:
I've tried to keep old code struct whenever the new SDK didn't enforce changes generating code in different places,
but that only to minimize changes and easy further code re-factoring: so feel free to move things